README.md

```diff
@@ -199,9 +199,11 @@ areas of the website:
 
 Use the `blog` archetype to create a new markdown content file in `content/blog`:
 
-    $ yarn hugp new --kind blog blog/URL-SLUG/index.md # replace URL-SLUG with URL slug for the blog post
+    $ yarn hugp new --kind blog blog/POST-HOME/index.md # replace POST-HOME with the pattern aaaa/mm/url-slug
```

Comment:
Perhaps we can make it a bit more clear:
```suggestion
    $ yarn hugp new --kind blog blog/YYYY/MM/index.md # replace YYYY with the year and MM with the month
```

Reply:
Thanks. We also need the PostName in the URL scheme in addition to YYYY and MM.